engine-tests/src/tests/erc20_connector.rs

```diff
@@ -482,6 +494,104 @@ pub mod sim_tests {
         );
     }
 
+    #[test]
+    fn test_exit_to_near_wnear_unwrapped() {
+        // Deploy Aurora; deploy wnear; bridge wnear to ERC-20 on Aurora
+        let TestExitToNearContext {
+            ft_owner,
+            ft_owner_address,
+            aurora,
+            wnear,
+            wnear_erc20,
+            ..
+        } = test_exit_to_near_common();
+
+        let ft_owner_balance = get_account_balance(ft_owner.account_id.as_str(), &aurora);
+
+        // Call exit function on ERC-20; observe ERC-20 burned + near tokens transferred
+        let result = exit_to_near(
+            &ft_owner,
+            &format!("{}:unwrap", ft_owner.account_id.as_str()),
+            FT_EXIT_AMOUNT,
+            &wnear_erc20,
+            &aurora,
+        );
+        let total_tokens_burnt: u128 = result
+            .promise_results()
+            .iter()
+            .map(|r| r.as_ref().unwrap().outcome().tokens_burnt)
+            .sum();
+
+        // Check that the wnear tokens are properly unwrapped and transferred to `ft_owner`
+        assert_eq!(
+            get_account_balance(ft_owner.account_id.as_str(), &aurora),
+            ft_owner_balance - total_tokens_burnt + FT_EXIT_AMOUNT
+        );
+
+        // Check wnear balances
+        assert_eq!(
+            nep_141_balance_of(ft_owner.account_id.as_str(), &wnear, &aurora),
+            FT_TOTAL_SUPPLY - FT_TRANSFER_AMOUNT
+        );
+        assert_eq!(
+            nep_141_balance_of(aurora.contract.account_id.as_str(), &wnear, &aurora),
+            FT_TRANSFER_AMOUNT - FT_EXIT_AMOUNT
```

Comment:
I was a bit confused about this constant due to its naming. I know this PR didn't introduce it, so just commenting: would it be more accurate to name this constant `FT_ERC20_AMOUNT`?

Reply:
To be honest, this is a subjective perception, the `FT_ERC20_AMOUNT` is also a little bit confusing because it sounds like the origin of the token is a native erc20 token that deployed on near.